srv/README.md

```diff
@@ -1,6 +1,19 @@
 # Demo server
 
-## Getting started
+## Getting started (Docker Compose)
+
+1. From inside the `/srv` directory, run Docker Compose as follows:
+
+```shell
+docker-compose up
+```
+
+2. Access the web server at `http://localhost:23500/api/dashboard/poll`
+
+3. Stop the container with CTRL+C.
+
+
+## Getting started (Docker)
```

Comment:
Sorry, I must have done something wrong with the rebase. Your commits should not show up in this pull request. Sorry, please ignore your code this time.